S7.Net/PLC.cs

```diff
@@ -153,6 +159,121 @@ private ErrorCode Connect(Socket socket)
             return LastErrorCode;
         }
 
+        /// <summary>
+        /// Describes a TPKT Packet
+        /// </summary>
+        private class TPKT
```

Comment:
I'm not a fan of  private classes. Can you move these to internal classes in their own files ?

Reply:
Agreed 😉